spec/factories/user_factory.rb

```diff
@@ -9,5 +9,6 @@
     password { Faker::Internet.password }
     role
     last_login { nil }
+    language { Faker::Internet.domain_suffix }
```

Comment:
A bit of an odd choice to choose for the value. If Faker doesn't have anything, I would just do `[en, fr, es, ar].sample` or something along that line